### PR TITLE
Staff Update

### PR DESCRIPTION
**Pull request by Inscrutable**
Dual is elevated, blood is retired.
I am  making this a PR to let others review, in case we have any other changes to make.

### CODE REVIEW

**Comment by Spongy** [comment]
A preview for this pull request is available at https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/59d1224/.

Here are some links to the pages that were modified:

- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/59d1224/about/staff.html

_Since the preview frequently changes, please link to [this comment](https://github.com/SpongePowered/SpongeDocs/pull/872#issuecomment-855348844), not to the direct url to the preview._

File: source/about/staff.rst [hunk 1]
```diff
@@ -13,11 +13,11 @@ operates efficiently.
 +-----------------------------------+------------------------------------+
 | Name                              | Role                               |
 +===================================+====================================+
-| blood                             | Implementation (Forge) Leader      |
+| Dualspiral                        | Sponge Leader                      |
```
Comment (kashike) [inline source/about/staff.rst]: Why uppercase name?

File: source/about/staff.rst [hunk 2]
```diff
@@ -13,11 +13,11 @@ operates efficiently.
...
 +-----------------------------------+------------------------------------+
-| gabizou                           | API Design Specialist & Leader     |
+| Gabizou                           | Sponge Leader                      |
```
Comment (kashike) [inline source/about/staff.rst]: Why uppercase name?

File: source/about/staff.rst (hunk 1 above)
Comment (Inscrutable) [reply source/about/staff.rst]: Actually, I just tweaked Gabizou to match the other two Leaders. The rest was already that way. We have no conventions at all on nick capitalisation, so it's more a matter of personal preference.

File: source/about/staff.rst (hunk 2 above)
Comment (Inscrutable) [reply source/about/staff.rst]: Which do you prefer @gabizou ?

**Review by ST-DDT — APPROVED** [review]

File: source/about/staff.rst (hunk 1 above)
Comment (kashike) [reply source/about/staff.rst]: @dualspiral Which do you prefer?

Comment (dualspiral) [reply source/about/staff.rst]: All lowercase please.

**Review by gabizou — APPROVED** [review]

**Review by kashike — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Pull request merged**